### PR TITLE
Fix weight and undefined index caused by incomplete export

### PR DESCRIPTION
**Pull request by jrhodes-sitback**

### CODE REVIEW

File: config/optional/block.block.socialsharingblock.yml [hunk 1]
```diff
@@ -1,14 +1,17 @@
+uuid: 96c79161-dd60-4aa9-bd22-ef52fca03d6c
```
Comment (chinthakagodawita) [inline config/optional/block.block.socialsharingblock.yml]: Could you remove this from the export? It shouldn't be in default exports.

File: config/optional/block.block.socialsharingblock.yml [hunk 2]
```diff
@@ -1,14 +1,17 @@
...
 langcode: en
 status: true
 dependencies:
   module:
     - social_media
   theme:
     - bartik
+_core:
+  default_config_hash: ZT4GpiuERHcE52YunPd3wrMxx8BJwJMovYPjMPET5m8
```
Comment (chinthakagodawita) [inline config/optional/block.block.socialsharingblock.yml]: Remove this too

File: config/optional/social_media.settings.yml [hunk 3]
```diff
@@ -5,8 +5,11 @@ social_media:
     api_url: 'http://www.facebook.com/share.php?u=[current-page:url]&title=[current-page:title]'
     api_event: href
     default_img: 1
-    weight: 1
+    weight: '1'
     attributes: "target|_blank\r\nclass|facebook-share"
+    drupalSettings: ''
+    library: ''
+    img: ''
```
Comment (chinthakagodawita) [inline config/optional/social_media.settings.yml]: Remove these 3 lines unless we're doing something specific with those options.

File: config/optional/social_media.settings.yml [hunk 4]
```diff
@@ -24,22 +27,33 @@ social_media:
     api_event: href
     text: Linkedin
     default_img: 1
-    weight: 3
+    weight: '3'
     attributes: "target|_blank\r\nclass|linkedin"
+    drupalSettings: ''
+    library: ''
+    img: ''
```
Comment (chinthakagodawita) [inline config/optional/social_media.settings.yml]: Remove these 3 lines too.

File: config/optional/social_media.settings.yml [hunk 5]
```diff
@@ -15,7 +18,7 @@ social_media:
     library: social_media/facebook
     text: 'Facebook messenger'
     default_img: 1
-    weight: 2
+    weight: '2'
```
Comment (chinthakagodawita) [inline config/optional/social_media.settings.yml]: Could you change this to an integer?

File: config/optional/social_media.settings.yml [hunk 6]
```diff
@@ -24,22 +27,33 @@ social_media:
     api_event: href
     text: Linkedin
     default_img: 1
-    weight: 3
+    weight: '3'
```
Comment (chinthakagodawita) [inline config/optional/social_media.settings.yml]: Could you change this to an integer?

File: config/optional/social_media.settings.yml [hunk 7]
```diff
@@ -5,8 +5,11 @@ social_media:
     api_url: 'http://www.facebook.com/share.php?u=[current-page:url]&title=[current-page:title]'
     api_event: href
     default_img: 1
-    weight: 1
+    weight: '1'
```
Comment (chinthakagodawita) [inline config/optional/social_media.settings.yml]: Could you change this to an integer?

File: config/optional/social_media.settings.yml [hunk 8]
```diff
@@ -24,22 +27,33 @@ social_media:
...
     default_img: 1
-    weight: 5
+    weight: '5'
     attributes: class|email
+    drupalSettings: ''
+    library: ''
+    img: ''
 langcode: en
+_core:
+  default_config_hash: ijnKXqWRB4kppnr81AwvgimHom2cxgEO4qTWDKxud84
```
Comment (chinthakagodawita) [inline config/optional/social_media.settings.yml]: Remove this.

File: config/optional/social_media.settings.yml [hunk 9]
```diff
@@ -24,22 +27,33 @@ social_media:
...
+    library: ''
+    img: ''
   email:
     enable: 1
     api_url: 'mailto:?subject=[current-page:title]&body=Check out this site [current-page:url]'
     api_event: href
     text: Email
     default_img: 1
-    weight: 5
+    weight: '5'
```
Comment (chinthakagodawita) [inline config/optional/social_media.settings.yml]: Could you change this to an integer?

File: config/optional/social_media.settings.yml [hunk 10]
```diff
@@ -24,22 +27,33 @@ social_media:
...
+    library: ''
+    img: ''
   twitter:
     enable: 1
     api_url: 'https://twitter.com/intent/tweet?url=[current-page:url]&status=[current-page:title]+[current-page:url]'
     api_event: href
     text: Twitter
     default_img: 1
-    weight: 4
+    weight: '4'
```
Comment (chinthakagodawita) [inline config/optional/social_media.settings.yml]: Could you change this to an integer?

File: config/optional/social_media.settings.yml [hunk 11]
```diff
@@ -24,22 +27,33 @@ social_media:
...
     attributes: "target|_blank\r\nclass|twitter"
+    drupalSettings: ''
+    library: ''
+    img: ''
```
Comment (chinthakagodawita) [inline config/optional/social_media.settings.yml]: Remove these 3 lines.

File: config/optional/social_media.settings.yml [hunk 12]
```diff
@@ -24,22 +27,33 @@ social_media:
...
     api_url: 'mailto:?subject=[current-page:title]&body=Check out this site [current-page:url]'
     api_event: href
     text: Email
     default_img: 1
-    weight: 5
+    weight: '5'
     attributes: class|email
+    drupalSettings: ''
+    library: ''
+    img: ''
```
Comment (chinthakagodawita) [inline config/optional/social_media.settings.yml]: Remove these 3 lines.

File: config/optional/social_media.settings.yml (hunk 5 above)
Comment (jrhodes-sitback) [inline config/optional/social_media.settings.yml]: Done

File: config/optional/social_media.settings.yml (hunk 6 above)
Comment (jrhodes-sitback) [inline config/optional/social_media.settings.yml]: Done

File: config/optional/social_media.settings.yml (hunk 4 above)
Comment (jrhodes-sitback) [inline config/optional/social_media.settings.yml]: These are needed as the code checks these values and throws warnings if not set.

File: config/optional/social_media.settings.yml (hunk 10 above)
Comment (jrhodes-sitback) [inline config/optional/social_media.settings.yml]: Done

File: config/optional/social_media.settings.yml (hunk 11 above)
Comment (jrhodes-sitback) [inline config/optional/social_media.settings.yml]: These are needed as the code checks these values and throws warnings if not set.

File: config/optional/social_media.settings.yml (hunk 9 above)
Comment (jrhodes-sitback) [inline config/optional/social_media.settings.yml]: Done

File: config/optional/social_media.settings.yml (hunk 12 above)
Comment (jrhodes-sitback) [inline config/optional/social_media.settings.yml]: These are needed as the code checks these values and throws warnings if not set.

File: config/optional/social_media.settings.yml (hunk 8 above)
Comment (jrhodes-sitback) [inline config/optional/social_media.settings.yml]: Done

File: config/optional/block.block.socialsharingblock.yml (hunk 2 above)
Comment (jrhodes-sitback) [inline config/optional/block.block.socialsharingblock.yml]: Done

File: config/optional/block.block.socialsharingblock.yml (hunk 1 above)
Comment (jrhodes-sitback) [inline config/optional/block.block.socialsharingblock.yml]: Done

**Pull request merged**